caffe2/proto/torch.proto

```diff
@@ -383,6 +421,17 @@ message TensorProto {
   // When this field is present, the data_type field MUST be
   // UINT32 or UINT64
   repeated uint64 uint64_data = 11 [packed = true];
+
+  // External data by file name
+  optional string external_data = 13;
+
+  // If two tensors represent the same weights/content, use alias.
+  // Must exist a TensorProto named alias in the initializer list.
+  // To avoid the duplicate tensor in attribute, such as value in Constant node.
+  optional string alias = 16;
```

Comment:
Is this necessary? If external_data is the same for two tensors, then we know share the same storage.

Reply:
This provide another indirection if we want to save everything in one file :-)